swagger/body-complex.json

```diff
@@ -20,6 +20,11 @@
       "get": {
         "operationId": "basic_getValid",
         "description": "Get complex type {id: 2, name: 'abc', color: 'YELLOW'}",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
```

Comment:
This should not be needed, since actually the server does nothing with ApiVersion. Why do you think it's important?

Reply:
Ah I see, I did not know that. I was looking at the other test swaggers for the testserver and noticed that all of the swaggers that specify ApiVersion also specify it for the other related GET, DELETE, POST, PUT requests (like complex-model.json, storage.json, subscriptionId-apiVersion.json and validation.json). But I can remove that change if you guys think it's not necessary in this case.